examples/WebApi/Controllers/HandleIncomingIceEventController.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.AspNetCore.Mvc;
+using Sinch;
+using Sinch.Voice.Calls.Actions;
+using Sinch.Voice.Calls.Instructions;
+using Sinch.Voice.Hooks;
+
+namespace WebApiExamples.Controllers
+{
+    [ApiController]
+    [Route("voice")]
+    public class HandleIncomingIceEventController : ControllerBase
+    {
+        private readonly ISinchClient _sinchClient;
+
+        public HandleIncomingIceEventController(ISinchClient sinchClient)
+        {
+            _sinchClient = sinchClient;
+        }
+
+        [HttpPost]
+        [Route("event")]
+        public IActionResult HandleEvent([FromBody] IVoiceEvent incomingEvent)
+        {
+            switch (incomingEvent)
```

Comment:
Ideally Voice should provide a `ValidateAuthenticationHeader` function (like Conversation domains: https://github.com/sinch/sinch-sdk-dotnet/blob/main/src/Sinch/Conversation/Webhooks/Webhooks.cs#L72)

Calling `ValidateAuthenticationHeader` here and because this sample is a tutorial, it will help to highlight endpoint is having to be "protected" against unsolicited requests

Reply:
Makes sense, blocked by #62